### PR TITLE
Edu Programs Migrations

### PR DESCRIPTION
**Pull request by devin-mccurdy**
Database migration PR for https://github.com/department-of-veterans-affairs/gibct-data-service/pull/445

### CODE REVIEW

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: db/migrate/20190923162900_create_edu_programs.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      t.string :student_vet_group
+      t.string :student_vet_group_website, null: false
+      t.string :vet_success_name, null: false
+      t.string :vet_success_email, null: false
+      t.string :vet_tec_program
+      t.integer :tuition_amount, null: false
+      t.integer :length_in_weeks, null: false
+    end
+
+    add_index :edu_programs, [:facility_code, :vet_tec_program] , using: :btree
```
Comment (annaswims) [inline db/migrate/20190923162900_create_edu_programs.rb]: will you generally be querying on `LOWER(vet_tec_program)` like here? if so, you may want to make  alter the index. https://github.com/department-of-veterans-affairs/gibct-data-service/pull/445/files#diff-79373e1323911953ef2804c7b52cebb8R554

**Review by annaswims — APPROVED** [review]
Please beware that migrations are not automatically run on deploys. So, this will need to be merged, deployed and had migrations run before we deploy the PR that depends on these changes.

**Pull request merged**